doc/tor.md

```diff
@@ -211,3 +209,103 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world usage of the Tor network is monitored and/or blocked.
+  If you are in an environment that does not permit direct Tor connections or the use
+  of Tor bridges, then considering the trade-offs, it may not be safe to use Tor. Other
+  options for privacy in Bitcoin Core: i2p.
+
+- For maximum privacy, it is preferable to disable listening instead of running an
+  onion service.
+
+- Users can download packages with [torsocks](https://gitlab.torproject.org/legacy/trac/-/wikis/doc/torsocks) for better security and privacy.
+
+- Trade-offs involved in using `onlynet=onion`:
+
+  Placing onlynet=onion in your bitcoin.conf file will ensure that your node only attempts to connect over
```

Comment:
```suggestion
  The `onlynet=onion` configuration option can potentially ensure the node only attempts to connect over
```

(note, config options may be used on the command line as well as in the conf file)